submitter/adaptors/terraform_adaptor.py

```diff
@@ -120,12 +124,15 @@ def __init__(self, adaptor_id, config, dryrun, validate=False, template=None):
         self.vars_file_tmp = "{}terraform.tfvars.json.tmp".format(self.volume)
         self.account_file = "{}accounts.json".format(self.volume)
         self.oci_auth_key = "{}oci_api_key.pem".format(self.volume)
-        self.configure_file = "{}configure".format(self.volume)
+        self.configure_file = "{}configure.py".format(self.volume)
         self.token_file = "{}pyGetScopedToken.py".format(self.volume)
+        self.preprocess_file = "{}preprocess.py".format(self.volume)
```

Comment:
So each time a new pre-processing script is added to MiCADO, we have to change the Terraform Adaptor as well? Does that not defeat the purpose of having  the wrapper?

Reply:
Currently from what I have done this name will be same for whichever cloud provider you are using. Just that the contents will be different to suit each cloud provider.